src/main/java/se/sundsvall/garbage/api/GarbageResource.java

```diff
@@ -28,17 +29,18 @@
 @ApiResponse(responseCode = "500", description = "Internal Server Error", content = @Content(schema = @Schema(implementation = Problem.class)))
 @Tag(name = "GarbageSchedule")
 public class GarbageResource {
-    
-    private final GarbageService garbageService;
-    
-    public GarbageResource(GarbageService garbageService) {
-        this.garbageService = garbageService;
-    }
-    
-    @GetMapping
-    @Operation(summary = "Get garbage schedule for an address")
-    @ApiResponse(responseCode = "200", description = "Successful Operation", content = @Content(schema = @Schema(implementation = GarbageScheduleResponse.class)))
-    public ResponseEntity<List<GarbageScheduleResponse>> getGarbage(@ParameterObject @Valid GarbageScheduleRequest request) {
-        return ResponseEntity.ok(garbageService.getGarbageSchedules(request));
-    }
+
+	private final GarbageService garbageService;
+
+	public GarbageResource(final GarbageService garbageService) {
+		this.garbageService = garbageService;
+	}
+
+	@GetMapping
+	@Operation(summary = "Get garbage schedule for an address")
+	@ApiResponse(responseCode = "200", description = "Successful Operation", content = @Content(schema = @Schema(implementation = GarbageScheduleResponse.class)))
```

Comment:
Change schema for 400 to `schema = @Schema(oneOf = {Problem.class, ConstraintViolation.class})`?